orchagent/eliminate_events.lua

```diff
@@ -0,0 +1,63 @@
+-- KEYS - None
+-- ARGV - None
```

Comment:
lua should be running only if the feature is enabled.

Reply:
It also needs to fetch the configuration from the database during initialization.